components/trace/appmap/index.test.mjs

```diff
@@ -11,7 +11,10 @@ const { undefined } = globalThis;
 const configuration = extendConfiguration(
   createConfiguration("protocol://host/home/"),
   {
-    name: "name",
+    appmap_dir: "appmap_dir",
+    appmap_file: "appmap_file",
+    "map-name": "map-name",
```

Comment:
This is a rather unfortunate key name. Is it dictated by some external constraint?